helm-charts/codegen/Chart.yaml

```diff
@@ -17,25 +17,32 @@ dependencies:
   - name: llm-uservice
     version: 0-latest
     repository: "file://../common/llm-uservice"
+    condition: llm-uservice.enabled
   - name: tei
     version: 0-latest
     repository: "file://../common/tei"
+    condition: tei.enabled
```

Comment:
Do we plan to disable this tei? If so, please make sure it's explicitly enabled in default settings a.k.a. `values.yaml`